.ci_scripts/install_dependencies.bat

```diff
@@ -0,0 +1,8 @@
+cd ..
+mkdir tmp
+cd tmp
+curl -o SDL2-devel-2.0.12-VC.zip https://www.libsdl.org/release/SDL2-devel-2.0.12-VC.zip
```

Comment:
Why 2.0.12? SDL is currently at version 2.0.16 already.

Reply:
Done